Reloj_checador/app/assets/stylesheets/home.scss

```diff
@@ -68,13 +68,12 @@ html,body{
     
     .login_btn{
     color: black;
-    background-color: #FFC312;
     width: 100px;
+    background-color: white !important;
```

Comment:
Using `!important` may cause issues if you want to override styles later on. Is it really needed here? Maybe this works without it. 🤔